src/components/WalletInfo/WalletInfo.scss

```diff
@@ -0,0 +1,31 @@
+.wallet-info {
+  display: flex;
+  align-items: center;
+
+  &--is-menu {
+    background-color: var(--c-grey);
+    border: .06rem solid var(--c-dark-border);
+    border-radius: .3rem;
+  }
+  &--is-banner {
+    background-color: transparent;
+  }
+  &--padding {
```

Comment:
https://github.com/airswap/airswap-marketplace/blob/main/CONTRIBUTING.md#naming-of-modifiers